markdown/pom.xml

```diff
@@ -54,6 +55,12 @@
       <version>${pegdown.version}</version>
     </dependency>
 
+    <dependency>
+       <groupId>com.vladsch.flexmark</groupId>
+       <artifactId>flexmark-all</artifactId>
+       <version>${flexmark.all.version}</version>
```

Comment:
What is the license of flexmark-all ? You need to update `LICENCE` file located in zeppelin root folder.